transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_prod_performance_events.sql

```diff
@@ -0,0 +1,13 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['received_at_date'],
```

Comment:
Looking at how long it takes to run on `TRANSFORM_XS`. Thinking it makes sense to run this on `TRANSFORM_L`.

Reply:
Yes, I'm also seeing this

Reply:
Can you share the query id? Thanks.

Reply:
I shared the ID with Catalin, marking this as resolved.